docs/docs/integrations/libraries/dbt/index.md

```diff
@@ -12,32 +12,117 @@ canonicalUrl: '/integrations/libraries/dbt'
 
 - Use Dagster's UI or APIs to run subsets of your dbt models, seeds, and snapshots.
 - Track failures, logs, and run history for individual dbt models, seeds, and snapshots.
 - Define dependencies between individual dbt models and other data assets. For example, put dbt models after the Fivetran-ingested table that they read from, or put a machine learning after the dbt models that it's trained from.
 
-## Installation
+## 1. Prepare a Dagster project
+
+To begin, you'll need a Dagster project. You can use an [existing components-ready project](/guides/build/projects/moving-to-components/migrating-project) or create a new one:
+
+<CliInvocationExample path="docs_snippets/docs_snippets/guides/components/integrations/dbt-component/1-scaffold-project.txt" />
+
+Activate the project virtual environment:
+
+<CliInvocationExample contents="source .venv/bin/activate" />
+
+Then, add the `dagster-dbt` library to the project, along with a duckdb adapter:
+
+<PackageInstallInstructions packageName="dagster-dbt dbt-duckdb" />
+
+## 2. Set up a dbt project
+
+For this tutorial, we'll use the jaffle shop dbt project as an example. Clone it into your project:
+
+<CliInvocationExample path="docs_snippets/docs_snippets/guides/components/integrations/dbt-component/3-jaffle-clone.txt" />
+
+We will create a `profiles.yml` file in the `dbt` directory to configure the project to use DuckDB:
+
+<CodeExample
+  path="docs_snippets/docs_snippets/guides/components/integrations/dbt-component/4-profiles.yml"
+  title="dbt/profiles.yml"
+  language="yaml"
+/>
+
+## 3. Scaffold a dbt component
+
+Now that you have a Dagster project with a dbt project, you can scaffold a dbt component. You'll need to provide the path to your dbt project:
+
+<CliInvocationExample path="docs_snippets/docs_snippets/guides/components/integrations/dbt-component/5-scaffold-dbt-component.txt" />
+
+The scaffold call will generate a `defs.yaml` file in your project structure:
+
+<CliInvocationExample path="docs_snippets/docs_snippets/guides/components/integrations/dbt-component/6-tree.txt" />
+
+In its scaffolded form, the `defs.yaml` file contains the configuration for your dbt project:
+
+<CodeExample
+  path="docs_snippets/docs_snippets/guides/components/integrations/dbt-component/7-component.yaml"
+  title="my_project/defs/dbt_ingest/defs.yaml"
+  language="yaml"
+/>
+
+This is sufficient to load your dbt models as assets. You can use `dg list defs` to see the asset representation:
+
+<WideContent maxSize={1100}>
+  <CliInvocationExample path="docs_snippets/docs_snippets/guides/components/integrations/dbt-component/8-list-defs.txt" />
+</WideContent>
+
+## 4. Run your dbt models
+
+To execute your dbt models, you can use the `dg launch` command to kick off a run through the CLI:
+
+<CliInvocationExample path="docs_snippets/docs_snippets/guides/components/integrations/dbt-component/9-dbt-run.txt" />
```

Comment:
````suggestion
<CliInvocationExample path="docs_snippets/docs_snippets/guides/components/integrations/dbt-component/9-dbt-run.txt" />

:::tip

You can also pass an [asset selection](https://docs.dagster.io/guides/build/assets/asset-selection-syntax) to the [dg launch --assets](https://docs.dagster.io/api/clis/dg-cli/dg-cli-reference#dg-launch) command:

```bash
dg launch --assets "key:'customers' and key:'orders'"
```
:::
````

Reply:
(Feel free to put a different example asset selection in here, I just picked two assets from the table above)